code/game/objects/structures/fluff.dm

```diff
@@ -263,3 +271,221 @@
+/obj/structure/fluff/haunted_tv
+	name = "Television"
+	desc = "Hey, where did the signal go?"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "haunted_tv"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/medieval_stage1
+	name = "Anvil"
+	desc = "Top of the line technology from around ten centuries ago."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "anvil"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/medieval_stage2
+	name = "Forge"
+	desc = "Top of the line technology from around ten centuries ago."
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "furnace_on"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/skeletal_bodybody
+	name = "remains"
+	desc = "A fake pile of bones and a skull. Spooky!"
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
+	layer = 3
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = FALSE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/skeletal_showcase
+	name = "skeleton showcase"
+	desc = "A totally fake replica of a spooky skeleton! On second thought, wait, is it fake?"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "showcase_6"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/unfinished_placeholder // meant so I can reveal a missing structure more easily.
+	name = "FINISH THIS LATER"
+	desc = "guys, someone code it in pls"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "error" // don't fix the error on purpose
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+/obj/structure/fluff/fakespace
+	name = "space"
+	desc = ""
+	icon = 'icons/turf/space.dmi'
+	icon_state = "0"
+	mouse_opacity = 0
+	layer = 5
+
+/obj/structure/fluff/starspawn
+	name = "meta-transdimensional rift"
+	desc = "<span class='nicegreen'>What mindfuckery is this?! Agh, I'm going MAD! I'M GOING INSANE!!! AAAAAAAAARGH!!!!!</span>"
+	icon = 'icons/misc/tourny_items.dmi'
+	icon_state = "portal"
+	max_integrity = 9999
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	density = TRUE
+	anchored = TRUE
+	deconstructible = FALSE
+
+
+/obj/structure/fluff/narsie
+	name = "ritual rune"
+	desc = "Is that a rune of the Geometer? Man, I really hope fake runes don't count for rituals..."
```

Comment:
make this a real rune you cowards

you could make this a subtype of that failure rune that does nothing but damage the cultists who invoke it